stories/Button/Button.regression-test.story.jsx

```diff
@@ -138,3 +138,32 @@ ErrorStroked.args = {
   ...ErrorDefault.args,
   stroked,
 };
+
+export const WithIcon = Template.bind({});
+WithIcon.args = {
+  icon: 'info',
+};
+
+const TemplateIconOnly = args => <Button icon="info" {...args} />;
+
+export const IconOnly = TemplateIconOnly.bind({});
+
+export const IconOnlySmall = TemplateIconOnly.bind({});
+IconOnlySmall.args = {
+  size: 'small',
+};
+
+export const IconOnlyXSmall = TemplateIconOnly.bind({});
+IconOnlyXSmall.args = {
+  size: 'xsmall',
+};
+
```

Comment:
Since we are here, should we include a test for medium size to?

Reply:
I didn't get it man, medium size is yhe default value.

Reply:
My bad, I was looking only what you changed in the tab files changed